pallets/rolldown/src/lib.rs

```diff
@@ -311,6 +311,50 @@ pub mod pallet {
 			Ok(().into())
 		}
 
+		#[pallet::call_index(3)]
+		#[pallet::weight(T::DbWeight::get().reads_writes(1, 1).saturating_add(Weight::from_parts(40_000_000, 0)))]
+		pub fn withdraw(
+			origin: OriginFor<T>,
+			tokenAddress: [u8; 20],
+			amount: u128,
+		) -> DispatchResultWithPostInfo {
+			let account = ensure_signed(origin)?;
+
+			let eth_token_address: [u8; 20] = array_bytes::hex2array::<_, 20>(tokenAddress.clone())
```

Comment:
tjoken address is already of a type, convertion is not needed

Reply:
done